mujoco_py/mjrendercontext.pyx

```diff
@@ -250,6 +256,8 @@ cdef class MjRenderContext(object):
 
         self._scn.ngeom += 1
 
+    def add_pixels(self, *pixels_params):
```

Comment:
similar to comment above, maybe rename to `add_pixel_buffer` or `add_image_overlay`.

Reply:
please also add docstring